rest/admin_api.go

```diff
@@ -532,8 +531,7 @@ func (h *handler) handlePutDbConfig() (err error) {
 				return nil, err
 			}
 
-			headerVersion := h.rq.Header.Get("If-Match")
-			if headerVersion != "" && headerVersion != bucketDbConfig.Version {
+			if h.headerDoesNotMatchEtag("If-Match", bucketDbConfig.Version) {
```

Comment:
I know this is existing code - but I wonder if we should move `"If-Match"` into the function, as I don't see a case where we'd want the header name to be anything different.